src/ia-account-settings.ts

```diff
@@ -0,0 +1,920 @@
+              Change Internet Archive / Open Library password
+            </label>
+            <input
+              type="password"
+              class="form-control"
+              id="password"
+              name="password"
+              autocomplete="new-password"
+              @input=${this.setPassword}
+              @blur=${this.validatePassword}
+            />
+            <button
+              type="button"
+              class="password-icon"
+              @click="${(e: Event) => {
+                togglePassword(e, this.passwordField as HTMLInputElement);
+                this.showPassword = !this.showPassword;
+              }}"
+            >
+              ${this.showPassword ? eyeOpen : eyeCrossed}
+            </button>
+            <span class="error-field">${this.fieldsError.password}</span>
+          </div>
+
+          <div class="form-element">
+            <label>Set borrow history</label>
+            <input
+              type="checkbox"
+              id="loan-history"
+              name="loan-history"
+              .checked=${this.loanHistoryFlag === 'public' ||
+              this.loanHistoryFlag === true}
+              @click=${this.setLoanHistory}
+            />
+            <label for="loan-history"> Visible to the public</label>
+          </div>
+
+          <div class="form-element newsletter">
+            <label>Newsletter subscriptions</label>
+            <p>
+              Stay up to date with what's happening at the Internet Archive by
+              signing up for our free newsletters.
+            </p>
+            ${this.mailingListsTemplate}
+          </div>
+
+          <div class="form-element">
+            <label for="linked-account"
+              >Linked 3rd party accounts (e.g. Google)</label
+            >
+            ${Object.keys(this.linkedProviders).length
+              ? this.linkedAccountTemplate
+              : 'You have no linked accounts'}
+          </div>
+
+          <div
+            class="form-element delete-account ${this.attemptToDelete
+              ? 'hide'
+              : ''}"
+          >
+            <button
+              type="button"
+              class="delete-link"
+              href="javascript:void(0)"
+              @click=${() => {
+                this.attemptToDelete = true;
+              }}
+            >
+              Delete Internet Archive / Open Library account
+            </button>
+          </div>
+
+          ${this.attemptToDelete ? this.deleteAccountTemplate : ''}
+        </div>
+      </form>
+
+      ${this.userData.isAdmin ? this.adminFunctionsTemplate : ''}
+    </div>`;
+  }
+
+  get getResponseTemplate() {
+    if (this.responseFields?.success === false) {
+      return html`<span class="error-field"
+        >&#10006; ${this.responseFields?.error}</span
+      >`;
+    }
+
+    return Object.values(this.responseFields?.fields ?? {})?.map(
+      msg => html`<span class="success-field">&#10003; ${msg}</span>`
```

Comment:
Minor suggestion: Might be nice to just add a comment nearby noting what the entities `&#10003;` and `&#10006;` are, so that other maintainers don't have to look it up.